### PR TITLE
chore(gulpfile): add clean task

### PR DESCRIPTION
**Pull request by huguangju**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ node_modules
 .settings
 *.log
 dist/
+.idea
```
Comment (fesor) [inline .gitignore]: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (fesor) [inline .gitignore]: Please remove this. This should be handled by developer's global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (huguangju) [inline .gitignore]: Sorry.

Comment (fesor) [inline .gitignore]: Another note. You could use git revert to revert this changes :)

**Comment by fesor** [comment]
Could you squash commits?

**Comment by huguangju** [comment]
Already squashed. Thank you for reminding me.

**Comment by fesor** [comment]
Cool!
